drivers/docker/docklog/docker_logger_test.go

```diff
@@ -13,22 +14,34 @@ import (
 	"golang.org/x/net/context"
 )
 
+func testContainerDetails() (string, string, string) {
```

Comment:
I would consider returning `docker.PullImageOptions`?  Or consider using named returned variables, so the values are a bit more obvious